src/Tmds.Ssh/SftpClient.File.cs

```diff
@@ -145,12 +145,20 @@ public override ValueTask HandleResponse(SftpPacketType type, ReadOnlySequence<b
         {
             if (type != SftpPacketType.SSH_FXP_STATUS)
             {
-                _tcs.SetException(CreateExceptionForStatus(fields));
+                _tcs.SetException(CreateExceptionForUnexpectedType(type));
             }
             else
             {
-                // if (SSH_FXP_STATUS == success) else.....
-                _tcs.SetResult(true); // TODO parsing of SSH_FXP_STATUS
+                var statusTuple = ParseStatusFields(fields);
```

Comment:
I'm using the same coding conventions as .NET Core, which means only `var` when the type is explicit on the rhs (like constructor).
`(SftpErrorCode errorCode, string? errorMessage)`